### ISSUE TITLE
Example Questionnaire to render "clinical profile"

### ISSUE THREAD

**Issue by rahul1** [medplum/medplum#2755]
Recreate the left sidebar illustrated [here](https://www.medplum.com/docs/charting) (known as the "clinical profile") using Questionnaires. This is a common UI element in most EHRs, and we would like a demonstration of how to use our current components to build this in medplum.

The goal of building this as a Questionnaire is to avoid creation bespoke UI components for this use case. It is a non-goal do a pixel-perfect implementation of this mock-up. Rather, it is to capture the essence of this structure in way that is relatively plug and play for an existing app.

Depends on #2753

Proper ship vehicle for this is still TBD, but some options (in order of preference) are:

1. Storybook Story
2. FHIR bundle with Questionnaire
3. Sample Application

**Comment by codyebberson** [comment]
Fixed in https://github.com/medplum/medplum/pull/2999

Note that we did not use a `Questionnaire`, and instead used React directly.
